cmd/deploy/deploy.go

```diff
@@ -297,11 +312,19 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 		fmt.Sprintf("%s=%s", model.OktetoNamespaceEnvVar, okteto.Context().Namespace),
 	)
 
-	for _, command := range opts.Manifest.Deploy.Commands {
-		if err := dc.executor.Execute(command, opts.Variables); err != nil {
-			log.Infof("error executing command '%s': %s", command, err.Error())
-			return fmt.Errorf("error executing command '%s': %s", command, err.Error())
-		}
+	err = dc.deploy(opts)
+	if err != nil {
+		output = fmt.Sprintf("%s\nApp '%s' deployment failed: %s", output, opts.Name, err.Error())
```

Comment:
@rlamana what should we use here? `Repository`? `Application`?

Reply:
or nothing, we could just say `Deployment failed: ....`

Reply:
Deploy could/will work with multiple repositories described in the manifest, right? So the word in this case might be "Application"... don't you think?

Reply:
@rlamana today `Application` is not a concept in the UI, that would be a problem, no?

Reply:
Today, we use Git Repository, and Pipeline, and Stacks... we need to agree to one term, and start using it everywhere.

Reply:
Right now it's shown as a Pipeline, but we probably need to create a term and use it instead of the other ones